src/js/home.js

```diff
@@ -3,7 +3,7 @@ import { h } from 'preact'
 const Home = props => (
   <div>
     <p>Hello, this is the home page of the quiz portal.</p>
-    <p>Please login using the email <i>megh@gmail.com</i> and the password <i>password</i></p>
+    <p>Please login using the email <i>megh@gmail.com</i> and the password: <i>password</i></p>
```

Comment:
Please don't mix multiple concerns in a single commit, revert this temporarily in your next commit. Create a new branch from my master and and fix this in that branch. Later you can have file a PR for that fix. Though I dont think its  a good idea to pollute the commit history with such a trivial commit. Its legible anyways